rgw/v2/tests/s3_swift/configs/test_bucket_listing_flat_unordered.yaml

```diff
@@ -2,6 +2,7 @@ config:
  user_count: 1
  bucket_count: 1
  objects_count: 30
+ user_remove: true
```

Comment:
Can we by default remove created user along with data all the time so that we don't need to clean up any setup after any automation run.

Reply:
Hi Uday. Thanks for the review. 
Yeah, remove user and purge all respective data functionality is handled in some of the scripts. But wrt to multi-site fixes, yhe logic is to keep the user and its details so that the same can be tested with on the secpndary